sw/device/lib/crypto/drivers/kmac_test.c

```diff
@@ -0,0 +1,45 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/crypto/drivers/kmac.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+extern size_t nist_kmac_nr_of_vectors;
+extern kmac_test_vector_t* nist_kmac_vectors[];
+
+OTTF_DEFINE_TEST_CONFIG();
+bool test_main(void) {
+  
+  kmac_error_t err;
+  //kmac_csr_context_t csr_context;
+  uint32_t keccak_read_buf1[64];
+  uint32_t keccak_read_buf2[64];
+  uint8_t digest[250];
+  kmac_test_vector_t *current_test_vector;
+
+  // Test SHAKE128
```

Comment:
This includes SHA3 test vectors as well, right?

Reply:
Correct, I will fix the comment.